### ISSUE TITLE
Changing the Net to Resnet50

### ISSUE THREAD

**Issue by raymongL** [swuxyj/DeepHash-pytorch#1]
Thanks for sharing the code, beautifule work!

So i simply change the net to the pretrained Resnet50 of pytorch, and run DPSH and DHN on CUB-200-2011. But the mAP is poor, so i tried different learning rate, but not much of a change.

Any idea what's the problem? I modified the fc layer of Resnet 50 to hash code length and use the output to compute loss. Any other specific operations should be applied?

**Comment by swuxyj** [comment]
你可以参考一下这个链接去修改网络https://github.com/thuml/HashNet/blob/master/pytorch/src/network.py，但是我也没试过，按理说Resnet50特征提取能力更强效果会更好，也许是哪出错了再检查下呢😂

**Comment by rayLemond** [comment]
两年过去了… 依旧没能把DPSH和DHN在CUB上跑出稍微正常一点的结果…害

**Comment by KayeeWong** [comment]
> 两年过去了… 依旧没能把DPSH和DHN在CUB上跑出稍微正常一点的结果…害

现在有好结果了吗😂
